src/core/unit_tests/ParallelScriptInterface_test.cpp

```diff
@@ -96,35 +83,37 @@ BOOST_AUTO_TEST_CASE(ctor_dtor) {
   }
 
   /* Check that ctor and dtor were run on all nodes */
-  Testing::reduce_and_check(callbacks->comm(), TestClass::constructed);
```

Comment:
Are these changes intended to be part of this PR?

Reply:
Yes, I turned out that the test was broken.